src/index.ts

```diff
@@ -6,5 +6,6 @@ export {
   type ButtonResetProps,
 } from './components/Button.js'
 export { TextInput, type TextInputProps } from './components/TextInput.js'
+export { extractFrameMetaTags } from './utils/extractFrameMetaTags.js'
```

Comment:
just popping a space in-between here

```suggestion
export { TextInput, type TextInputProps } from './components/TextInput.js'

export { extractFrameMetaTags } from './utils/extractFrameMetaTags.js'
```